challenge-06/challenge-06.js

```diff
@@ -32,19 +33,28 @@ Crie uma função chamada `showTeamPosition` com as seguintes características:
     - Se não houver time para a posição passada, deve retornar a mensagem:
     "Não temos a informação do time que está nessa posição."
 */
-// ?
+function showTeamPosition(numero) {
+    return numero <= 5 ? 'O time que está em ' + numero + 'º lugar é o ' + teams[--numero] + '.' : 'Não temos a informação do time que está nessa posição.';
```

Comment:
-1
É quase isso xD

Só tem que verificar o seguinte: você está retornando se a posição passada por parâmetro for menor ou igual a `5`. Isso deixa brecha pra eu usar algo como `showTeamPosition(0)`, ou ainda `showTeamPosition(-1)` ;)

A chamada do array funciona, mas você só precisa entender o que ele está fazendo de verdade:
o `--numero` vai fazer duas coisas:
1. Atribuir à `numero` tudo o que ele tem menos 1;
2. Pegar a chave do array `teams` que é o novo valor de `numero`.

Essa atribuição não é necessária, você poderia usar somente `teams[ numero - 1 ]`.

Como eu disse: funciona, mas precisa estar ciente dos efeitos colaterias disso, pois se em algum outro momento você precisasse do valor de `numero` dentro da mesma função, esse valor já seria diferente ;)